bundles/mapping/mapmodule/plugin/pinchzoomreset/PinchZoomResetPlugin.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { ZoomOutOutlined } from '@ant-design/icons';
+import { MapModuleButton } from '../../MapModuleButton';
+import styled from 'styled-components';
+
+// Icon is too small with defaults (18x18px)
+const StyledButton = styled(MapModuleButton)`
+> span {
+    font-size: 22px;
+    max-height: 22px;
+    max-width: 22px;
+    > svg {
+        max-height: 22px;
+        max-width: 22px;
+    }
+}
+`;
+
+/**
+ * @class Oskari.mapframework.bundle.mapmodule.plugin.PinchZoomReset
+ *
+ * Displays a button to reset pinch zoom on top of the map.
+ *
+ * This solves the situation where the user has pinch zoomed in sot that no other elements besides the map itself are no longer accessible.
```

Comment:
```suggestion
 * This solves the situation where the user has pinch zoomed in so that no other elements besides the map itself are no longer accessible.
```